src/parameterizations/vertical/MOM_CVMix_KPP.F90

```diff
@@ -537,10 +534,6 @@ logical function KPP_init(paramFile, G, GV, US, diag, Time, CS, passive)
       'Friction velocity, u*, as used by [CVMix] KPP', 'm/s', conversion=US%Z_to_m*US%s_to_T)
   CS%id_buoyFlux = register_diag_field('ocean_model', 'KPP_buoyFlux', diag%axesTi, Time, &
       'Surface (and penetrating) buoyancy flux, as used by [CVMix] KPP', 'm2/s3', conversion=US%L_to_m**2*US%s_to_T**3)
-  CS%id_QminusSW = register_diag_field('ocean_model', 'KPP_QminusSW', diag%axesT1, Time, &
```

Comment:
Just curious where/how these diagnosrics are being handled now.

Reply:
There are three new per-tracer diagnostics, with ids stored in `tracer_type`:

1. `id_net_surfflux` replaces `id_QminusSW` for temperature and `id_netS` for salt
2.  `id_NLT_tendency` replaces `id_NLT_dTdt` for temperature and `id_NLT_dSdt` for salt
3.  `id_NLT_budget` replaces `id_NLT_temp_budget` for temperature and `id_NLT_saln_budget` for salt